Containers/nextcloud/Dockerfile

```
@@ -61,7 +61,7 @@ RUN set -ex; \
     \
 # pecl will claim success even if one install fails, so we need to perform each install separately
     pecl install APCu-5.1.21; \
-    pecl install memcached-3.1.5; \
+    pecl install memcached-3.2.0RC2; \
```

```suggestion
    pecl install memcached-3.2.0; \
```

Should already be released...